### PR TITLE
Django 3.0 compatibility plus updated tests

### PR DESCRIPTION
**Pull request by lingster**

### CODE REVIEW

**Comment by ababic** [comment]
Would be lovely to see this merged :) The CI failure appears to be unrelated.

File: tox.ini [hunk 1]
```diff
@@ -3,7 +3,8 @@ envlist =
     flake8
     py{36}-dj{20}-wt{23}
     py{36,37}-dj{21}-wt{24}
-    py{36,37,38}-dj{22}-wt{25,26,27,28,29}
+    py{36,37,38}-dj{22}-wt{25,26,27,28}
+    py{37,38}-dj{30}-wt{28,29}
```
Comment (ababic) [inline tox.ini:7]: @lingster This line should be changed to: `py{37,38}-dj{30}-wt{28,29,**210**}`

File: tox.ini [hunk 2]
```diff
@@ -19,13 +20,15 @@ deps =
...
     dj22: Django>=2.2,<2.3
+    dj30: Django>=3.0,<3.1
     wt23: wagtail>=2.3,<2.4
     wt24: wagtail>=2.4,<2.5
     wt25: wagtail>=2.5,<2.6
     wt26: wagtail>=2.6,<2.7
     wt27: wagtail>=2.7,<2.8
     wt28: wagtail>=2.8,<2.9
-    wt29: wagtail>=2.9,<2.10
+    wt29: wagtail>=2.9,<3.0
```
Comment (ababic) [inline tox.ini:30]: This line should stay as is, and the following line added instead:
```wt210: wagtail>=2.10,<2.11```

File: tox.ini (hunk 1 above)
Comment (lingster) [reply tox.ini]: done, I think at the time of these changes wagtail 2.10.x had not been released

File: tox.ini (hunk 2 above)
Comment (lingster) [reply tox.ini]: amended as requested

File: setup.py [hunk 3]
```diff
@@ -17,7 +17,7 @@
 
 
 install_requires = [
-    'wagtail>=2,<2.10'
+    'wagtail>=2,<3.0'
```
[the remark this answers is not in the document]
Comment (lingster) [reply setup.py]: amended as requested

**Comment by lingster** [comment]
@nnist Would you be able to help review and merge this PR?

**Comment by nnist** [comment]
@lingster Sure! I'll check this out today or tomorrow.

**Comment by nnist** [comment]
Alright, I've been testing this for a bit and it seems good to go. I'll see if I can get this released tomorrow. Thanks for the contribution! :)

**Pull request merged**